collection-scripts/gather

```diff
@@ -29,6 +29,9 @@ echo "${resources[@]}" | tr ' ' '\n' | xargs -t -I{} -P "${PROS}" --max-args=1 s
 
 "${DIR_NAME}"/vmConvertor
 
+# Collect instancetype and preferences
+"${DIR_NAME}"/gather_instancetypes
```

Comment:
The simple approach is to collect by default here, otherwise we could make it conditional on each VirtualMachine, checking for references to an instancetype and/or preference.

Reply:
I think it would be enough to have just the `gather_instancetypes` script and users would just use it as a targeted gathering script just like they do for [VM information](https://github.com/kubevirt/must-gather#targeted-gathering---vm-information) and [Images information](https://github.com/kubevirt/must-gather#targeted-gathering---images-information)

When someone wanted to collect that info they would execute:

`oc adm must-gather --image=quay.io/kubevirt/must-gather -- /usr/bin/gather_instancetypes`

Reply:
ACK for now I guess but by 4.13 we do plan on starting to replace the use of templates with a common set of instancetypes and preferences so this is likely to become more useful.